common/static/common/js/components/views/tabbed_view.js

```diff
@@ -89,33 +110,128 @@
                        if (!(Backbone.history.getHash() in this.urlMap)) {
                            this.setActiveTab(0);
                        }
+
                        return this;
                    },
 
-                   setActiveTab: function (index) {
+                   setActiveTab: function(index) {
                        var tabMeta = this.getTabMeta(index),
                            tab = tabMeta.tab,
                            tabEl = tabMeta.element,
                            view = tab.view;
+                       
                        // Hide old tab/tabpanel
-                       this.$('button.is-active').removeClass('is-active').attr('aria-expanded', 'false');
-                       this.$('.tabpanel[aria-expanded="true"]').attr('aria-expanded', 'false').addClass('is-hidden');
+                       this.$('button.is-active')
```

Comment:
Yeah, this file looks to have used two spaces previously. I'm torn between using four (my default) but having it look messy, or using two (native) and having it be more readable. Trying to avoid touching everything else. Thoughts?